services/oauth/v2/utils.go

```diff
@@ -0,0 +1,141 @@
+package v2
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"syscall"
+	"time"
+
+	jsoniter "github.com/json-iterator/go"
+	"github.com/tidwall/gjson"
+
+	"github.com/rudderlabs/rudder-go-kit/stats"
+	router_utils "github.com/rudderlabs/rudder-server/router/utils"
+	"github.com/rudderlabs/rudder-server/utils/misc"
+)
+
+var jsonfast = jsoniter.ConfigCompatibleWithStandardLibrary
+
+const (
+	REFRESH_TOKEN = "REFRESH_TOKEN"
+	// Identifier to be sent from destination(during transformation/delivery)
+	AUTH_STATUS_INACTIVE = "AUTH_STATUS_INACTIVE"
+	// Identifier for invalid_grant or access_denied errors(during refreshing the token)
+	REF_TOKEN_INVALID_GRANT = "ref_token_invalid_grant"
```

Comment:
It's idiomatic to define constants as MixedCaps.
```suggestion
	RefreshToken = "REFRESH_TOKEN"
	// Identifier to be sent from destination(during transformation/delivery)
	AuthStatusInactive = "AUTH_STATUS_INACTIVE"
	// Identifier for invalid_grant or access_denied errors(during refreshing the token)
	RefTokenInvalidGrant = "ref_token_invalid_grant"
```